x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/package-info.java

```diff
@@ -184,6 +184,19 @@
  *         looks ok.
  *     </li>
  *     <li>
+ *         Let's finish up the code by making the tests backwards compatible. Since this is a new
+ *         feature we just have to convince the tests not to run in a cluster that includes older
+ *         versions of Elasticsearch. We do that with a {@link org.elasticsearch.rest.RestHandler#supportedCapabilities capability}
+ *         on the REST handler. ESQL has a <strong>ton</strong> of capabilities so we list them
+ *         all in {@link org.elasticsearch.xpack.esql.action.EsqlCapabilities}. Add a new one
+ *         for your function. Now add something like {@code required_capability: my_function}
+ *         to all of your csv-spec tests. Run those csv-spec tests as integration tests to double
+ *         check that they run on the main branch. A spelling mistake here will disable them. Do:
```

Comment:
Nit: We currently check that the capability exists here: https://github.com/elastic/elasticsearch/blob/8df1c504bbb52b756438efddba9d6c44fc96121e/x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/CsvTests.java#L251-L255

So just running the "unit" csv tests should check that. As to simplify this paragraph a bit. (Unless it didn't work for some reason; in which case we should maybe check it)

Reply:
Is there an action item for me here? 🤔

Reply:
Nah. I pushed that one. I still double check out of paranoia.

Reply:
It's the last ~4 lines of this paragraph, that describe how to find if you mistyped the capability. As the CsvTests already checks for capabilities that don't exist, there should be no need to do that process.
You can run the tests, and it will fail if something doesn't exist:
![image](https://github.com/user-attachments/assets/0755674b-27eb-4111-aa5f-a6a7381adc53)

So I'd probably remove that part, as to make this section shorter, and as the tests already take care of it

Reply:
Sure. Just zap that part please. I added it when I was helping get us up to date.